### ISSUE TITLE
ZIPinstall thinks multiple .gpr.py files are invalid

### ISSUE THREAD

**Issue by emyoulation** [Taapeli/isotammi-addons#7]
The ZIPinstall seems to have a validation with a rule that a ZIP may not have more than one .gpr.py Gramps Plugin Registration file.  But that is supported by the Gramps registration... both as files registering multiple plugins and as separate registration files for each add-on in a single plug-in folder.

Example:
[D3 Ancestral and Descendant Charts](https://gramps-project.org/wiki/index.php/Addon:D3_Ancestral_and_Descendant_Charts)
Type: Web Report  - Create ancestral/descendant web based interactive D3 charts. 	
Author: [Matt Keenan](mailto:matt.keenan@gmail.com) 	
[D3Charts.addon.](https://github.com/gramps-project/addons/blob/master/gramps51/download/D3Charts.addon.tgz?raw=true)

The [CardView addon repository folder](https://github.com/cdhorn/CardView) is another good example. It has more than a half dozen add view modes and a Gramplet in a single archive. (But since it is in beta and not passed through Peer Review, you might not think of it as a valid test.) In this case, the GitHub system's Code button has a "Download ZIP" pop-up menu option that generates the ZIP file on demand. So it seemed like verifying a GitHub generated download would be a good thing.

**Comment by kkujansuu** [comment]
Yes, I am aware of the issue. My defense is that initially the Isotammi addons were intended only for the Finnish Isotammi users where we have some control on which addons are used. Specifically ZipInstall was intended to be an alternative way to distribute some of our addons to users - not as a general mechanism to install any addon.

Enhancing ZipInstall to support all features is not trivial.
